Mobile/starter_project_dark_knights/lib/features/onboarding/presentation/widgets/onboarding/separator.dart

```diff
@@ -0,0 +1,15 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/src/widgets/framework.dart';
+import 'package:flutter/src/widgets/placeholder.dart';
+
+class Separator extends StatelessWidget {
```

Comment:
This is unnecessary because you can call the sizedbox directly.

Reply:
Done

Reply:
Done